podium_api/devices.py

```diff
@@ -27,6 +30,12 @@ def make_device_update(
     Kwargs:
         name(str): Name of the device.
 
+        private(int): 1 if the device is private to the creating user

```

Comment:
How about accepting the conventional boolean (True | False) instead of 1 | 0?

Reply:
The server side code has trouble accepting JSON booleans (Ruby on Rails) so the API uses integers to indicate true / false. 

However, your suggestion is good, so I made the python API use boolean, and converts them to and from integer as needed.  

Update code to match this.